contracts/optimistic-ethereum/OVM/execution/OVM_ExecutionManager.sol

```diff
@@ -159,7 +164,8 @@ contract OVM_ExecutionManager is iOVM_ExecutionManager, Lib_AddressResolver {
         override
         public
     {
-        require(transactionContext.ovmNUMBER == 0, "Only be callable at the start of a transaction");
+        require(transactionContext.ovmNUMBER == DEFAULT_UINT256, "Only callable at the start of a transaction");
```

Comment:
After discussions with John, realizing that this should return, not revert, so that code contracts cannot maliciously surface a malformed (i.e. not from `_encodeRevertFlag`) revert value.